src/feature.cc

```diff
@@ -25,6 +25,7 @@ const Feature Feature::Experimental3mfExport("3mf-export", "Enable 3MF export.")
 const Feature Feature::ExperimentalSvgImport("svg-import", "Enable SVG import.");
 const Feature Feature::ExperimentalCustomizer("customizer", "Enable Customizer");
 const Feature Feature::ExperimentalInputDriver("input-driver", "Enable input drivers (requires restart)");
+const Feature Feature::Experimental3dPrint("3d-print", "Design upload to Print a Thing service (requires restart)");
```

Comment:
Does it require restart?

Reply:
Yes, for showing/hiding the icons (well, new file might be enough).

Reply:
ok, doesn't hurt to mention it